crates/sqlite/Cargo.toml

```diff
@@ -5,14 +5,12 @@ authors = { workspace = true }
 edition = { workspace = true }
 
 [dependencies]
-async-trait = { workspace = true }
 serde = { workspace = true }
 spin-factor-sqlite = { path = "../factor-sqlite" }
 spin-factors = { path = "../factors" }
-spin-locked-app = { path = "../locked-app" }
-spin-resource-table = { path = "../table" }
+# spin-locked-app = { path = "../locked-app" }
```

Comment:
oops I forgot to delete these lines

Reply:
nicely timed for one. minute. after I went "yeah looks fine" _grin_

Reply:
indeed, a close call